test/src/api/responder04Test.php

```diff
@@ -45,11 +42,11 @@ public function test_interface_game_00061() {
             array('value' => NULL, 'sides' => 20, 'skills' => array('Ornery', 'Fire'), 'properties' => array(), 'recipe' => 'oF(C)', 'description' => 'Ornery Fire Wildcard die', 'wildcardPropsArray' => array('type' => 'Wildcard', 'suit' => 'Spades', 'colour' => 'black', 'displayedValue' => 'J<span class="suit_black">&spades;</span>')),
         );
         $expData['playerDataArray'][1]['activeDieArray'] = array(
-            array('value' => NULL, 'sides' => 20, 'skills' => array(), 'properties' => array(), 'recipe' => '(C)', 'description' => 'Wildcard die', 'wildcardPropsArray' => array('type' => 'Wildcard', 'suit' => 'Clubs', 'colour' => 'black', 'displayedValue' => 'Q<span class="suit_black">&clubs;</span>')),
-            array('value' => NULL, 'sides' => 20, 'skills' => array(), 'properties' => array(), 'recipe' => '(C)', 'description' => 'Wildcard die', 'wildcardPropsArray' => array('type' => 'Wildcard', 'suit' => 'Spades', 'colour' => 'black', 'displayedValue' => '9<span class="suit_black">&spades;</span>')),
```

Comment:
Hmm.  If this passes now, i think it's not quite the right fix --- remember that during SPECIFY_DICE, you can't see your *own* die values either.  It's not just about hiding your opponent's die values --- knowing what you rolled might impact what swing/option die settings you want, so the API doesn't tell you what you rolled at that time.

Reply:
Ah right, easy enough to fix.